README.md

```diff
@@ -1,2 +1,33 @@
 # continunet
 Source finding package for radio continuum data powered by U-Net segmentation algorithm.
+
```

Comment:
I'd have a line here saying people can also just install with

```bash
pip install git+https://github.com/hstewart93/continunet
```

Put the easiest thing at the top, and the details elsewhere in the README.